docs/_guide/styles.md

```diff
@@ -133,14 +174,23 @@ This can be a good way to load CSS generated from tools like SASS/LESS.
 There are some important caveats though:
 
 * External styles can cause a flash-of-unstyled-content (FOUC) while they load.
-* The URL in the `href` attribute is relative to the _main document_, making this technique mostly useful for application elements where asset URLs are well known, and not for reusable elements published publically.
+* The URL in the `href` attribute is relative to the _main document_, making this technique mostly useful for application elements where asset URLs are well known, and not for reusable elements published publicly.
 
+### What you can style
 
-## Styling the host element
+You can style:
+
+* [The host element](#host). The host element is your component. It is called the "host" because it hosts shadow DOM. You can style the element itself with special CSS selectors `:host` and `:host()`.
+* [Elements in the host's shadow root](#children).
+* [Slotted elements](#slot). Slotted elements are rendered in shadow DOM via the `slot` element - see MDN for more info. You can style slotted content with `::slotted()`.
```

Comment:
```suggestion
* [Slotted elements](#slot). Slotted elements are light DOM children rendered in shadow DOM via the `slot` element - see MDN for more info. You can style slotted content with `::slotted()`.
```

Added "light dom children" event though that's not completely accurate because of re-projection, because I think the current phrasing doesn't differentiate from normal shadow DOM content. As far as most users are concerned (I think), the important part is that this is how you style children.